apple/RNCWebView.m

```diff
@@ -426,6 +426,8 @@ - (void)didMoveToWindow
     _webView.scrollView.pagingEnabled = _pagingEnabled;
     //For UIRefreshControl to work correctly, the bounces should always be true
     _webView.scrollView.bounces = _pullToRefreshEnabled || _bounces;
+    _webView.scrollView.alwaysBounceHorizontal = _pullToRefreshEnabled || _bounces;
+    _webView.scrollView.alwaysBounceVertical = _pullToRefreshEnabled || _bounces;
```

Comment:
While I trust that setting these to `false` works around the issue of setting `bounces` to `false`, I am concerned about side-effects, such as behavior on other iOS versions and setting these to `true` when `bounces` isn't `false` (e.g., according to Apple's documentation, the default value of `alwaysBounceVertical` is already supposed to be `false`)